lib/packages/crypto/README.md

```diff
@@ -0,0 +1,260 @@
+# @burstjs/crypto
+
+Cryptographic functions for building Burstcoin apps.
+
+## Installation
+
+Install using [npm](https://www.npmjs.org/):
+
+```
+npm install @burstjs/crypto
+```
+
+or using [yarn](https://yarnpkg.com/):
+
+``` yarn
+yarn add @burstjs/crypto
+```
+
+## API Reference
+## Classes
+
+<dl>
+<dt><a href="#BurstUtil">BurstUtil</a></dt>
+<dd><p>The BurstUtil class provides static methods for encoding and decoding numeric ids.
+In addition, addresses can be checked for validity.</p></dd>
+</dl>
+
+## Members
+
+<dl>
+<dt><a href="#BN">BN</a></dt>
+<dd><p>Original work Copyright (c) 2018 PoC-Consortium<br>Modified work Copyright (c) 2019 Burst Apps Team</p></dd>
+</dl>
+
+## Functions
+
+<dl>
+<dt><a href="#decryptAES">decryptAES(encryptedBase64, key)</a> ⇒</dt>
+<dd><p>Decrypt an encrypted message</p></dd>
+<dt><a href="#generateMasterKeys">generateMasterKeys(passPhrase)</a> ⇒</dt>
+<dd><p>Generate the Master Public Key and Master Private Key for a new passphrase</p></dd>
+<dt><a href="#generateSignature">generateSignature(messageHex, privateKey)</a></dt>
+<dd><p><p>Generate a signature for the transaction</p></p>
+<p><p>Method:</p></p>
+<pre class="prettyprint source"><code> s = sign(sha256(sha256(transactionHex)_keygen(sha256(sha256(transactionHex)_privateKey)).publicKey),
+         sha256(sha256(transactionHex)_privateKey),
+         privateKey)
+ p = sha256(sha256(transactionHex)_keygen(sha256(transactionHex_privateKey)).publicKey)</code></pre></dd>
+<dt><a href="#generateSignedTransactionBytes">generateSignedTransactionBytes(unsignedTransactionHex, signature)</a> ⇒</dt>
+<dd><p>Generates a signed message digest</p></dd>
+<dt><a href="#getAccountIdFromPublicKey">getAccountIdFromPublicKey(publicKey)</a> ⇒</dt>
+<dd><p>Convert hex string of the public key to the account id</p></dd>
+<dt><a href="#verifySignature">verifySignature(signature, messageHex, publicKey)</a></dt>
+<dd><p><p>Verify a signature for given message</p></p>
+<ul>
+<li>Method:<pre class="prettyprint source"><code>* h1 = sha256(sha256(transactionHex)_keygen(sha256(transactionHex_privateKey)).publicKey)
+==
+sha256(sha256(transactionHex)_verify(v, h1, publickey)) = h2</code></pre></li>
+</ul></dd>
+</dl>
+
+<a name="BurstUtil"></a>
+
+## BurstUtil
+<p>The BurstUtil class provides static methods for encoding and decoding numeric ids.
+In addition, addresses can be checked for validity.</p>
+
+**Kind**: global class  
+
+* [BurstUtil](#BurstUtil)
```

Comment:
We need to split the class in separate functions. Let's do this in another pr

Reply:
Agreed, I think this should be its own package tbh